checklist_plus/text_generation/masked_lm.py

```
@@ -1,335 +1,287 @@
+        options = all_possible_antonyms(word)
+        if synonym:
+            options = all_possible_synonyms(word)
+        if not options:
+            return []
+        options = options + [word]
+        masked = re.sub(r'\b%s\b' % re.escape(word), '[MASK]', text)
+        if masked == text:
+            return []
+        ret =  self.unmask(masked, beam_size=100000000, candidates=options)
+        non_word = [x for x in ret if np.all([y not in [self.tokenizer.unk_token, word] for y in x[0]])]
+        score = [x for x in ret if np.all([y in [word, self.tokenizer.unk_token] for y in x[0]])][0][-1]
```

Ruff (0.12.2)

Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)